openedx/core/djangoapps/site_configuration/management/commands/create_or_update_site_configuration.py

```diff
@@ -0,0 +1,97 @@
+"""
+Create or updates the SiteConfiguration for a site.
+"""
+from __future__ import unicode_literals
```

Comment:
Can you also add `absolute_import` too?

Reply:
I ran `pylint openedx/core/djangoapps/site_configuration/management/ --py3k` and it asked to add it. Doing so will likely make this easier to be accepted upstream